scripts/ceph-windows/setup_ceph_vstart

```diff
@@ -43,6 +45,13 @@ time rsync_cmd $WORKSPACE/ceph ${VSTART_DIR}/build-ceph-vstart.sh ${UBUNTU_SSH_U
 time SSH_TIMEOUT=1h ssh_exec ./build-ceph-vstart.sh
 ssh_exec sudo apt-get install -y python3-prettytable
 
+objectstore_args=""
+if [ $USE_MEMSTORE == "yes" ]; then
```

Comment:
Since `USE_MEMSTORE` is user input, it's best to quote (and use `[[` instead of `[`).